rancher/install-rancher.sh

```diff
@@ -241,23 +242,23 @@ ssh-keygen -f /root/.ssh/id_rsa -P "" -q
 cat /root/.ssh/id_rsa.pub >> /root/.ssh/authorized_keys
 
 # Install Docker
-curl https://releases.rancher.com/install-docker/19.03.sh | sh
+curl https://releases.rancher.com/install-docker/20.10.sh | sh
 
 ### Download dependency
 
 # rke
-wget https://github.com/rancher/rke/releases/download/v1.1.2/rke_linux-amd64
+wget https://github.com/rancher/rke/releases/download/v1.3.7/rke_linux-amd64
```

Comment:
How about choosing either curl or wget?

```suggestion
curl -LO https://github.com/rancher/rke/releases/download/v1.3.7/rke_linux-amd64
```